### DISCUSSION TITLE
How to Ask For Help With Capstones!

### DISCUSSION THREAD

**Discussion by ajawashington** [nss-evening-cohort-14/evening-client-side#268] [Capstones!] [answered]
RULES:
You have to answer ALL of the questions below to use the Capstones! Discussion
DO NOT ASK FOR HELP WITH CAPSTONES ON ANOTHER DISCUSSION TYPE

Your Name
Describe your issue
List any error messages that appear
What have you googled? (Paste links to the things you have found from your google search - You must include at least 3 links)
What have you tried? Tell us everything you have tried.
Any API Responses you are getting. 

Copy/paste link TO THE FILE on your branch or in your repo that you are having issues with **NO SCREEN SHOTS** 🥰
NOTE: We will not accept links to the whole repo. You must provide a link directly to the specific file
ONCE YOU GET AN ANSWER: mark the answer so that others can get the benefit of seeing the answer. It may very well be that you are asked to go back to your thread and update it with the answer if the instruction team helps you with your issue.

Either way, once you get a resolution either through the discussion or outside of it, you must update with the answer you get.

Did I mention DO NOT ASK FOR HELP ON ANOTHER DISCUSSION TYPE??

Give me a comment below letting us know that you understand these items or let us know in the comments if you have any questions.

**Comment by chiestroud** [comment, marked as answer]
Understood!

**Comment by cmeffley** [comment]
Got it!

**Comment by Gabrielle-Tobermann** [comment]
👍

**Comment by tsekeres** [comment]
Right on.

**Comment by seancrossettie** [comment]
Got it!

**Comment by jimguistical** [comment]
Understood!
